src/Microsoft.Health.Fhir.Core/Features/Operations/Reindex/Models/ReindexJobRecord.cs

```diff
@@ -82,6 +86,18 @@ protected ReindexJobRecord()
         [JsonProperty(JobRecordProperties.MaximumNumberOfResourcesPerQuery)]
         public uint MaximumNumberOfResourcesPerQuery { get; private set; }
 
+        /// <summary>
+        /// Controls the time between queries of resources to be reindexed
+        /// </summary>
+        [JsonProperty(JobRecordProperties.QueryDelayIntervalInMilliseconds)]
+        public int QueryDelayIntervalInMilliseconds { get; set; }
+
+        /// <summary>
+        /// Controls the target percentage of how much of the allocated
+        /// data store resources to use
+        /// </summary>
+        public ushort? TargetDataStoreResourcePercentage { get; set; }
```

Comment:
I was also thinking through if the term "Usage" or "Consumption" would be better than "Resource" here?